components/automate-gateway/api/iam/v2beta/common/rules.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package chef.automate.api.iam.v2beta;
+option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2beta/common";
+
+enum RuleType {
```

Comment:
thanks for splitting this out! we probably should do the same for roles one of these days